core/src/services/azdls/core.rs

```diff
@@ -342,8 +345,14 @@ impl AzdlsCore {
         if let Some(limit) = limit {
             url = url.push("maxResults", &limit.to_string());
         }
+
         if !continuation.is_empty() {
             url = url.push("continuation", &percent_encode_path(continuation));
+        } else if let Some(start_after) = start_after {
+            url = url.push(
+                "startAfter",
```

Comment:
Does azdls really has a query param called `startAfter`? 

From [the hadoop code](https://github.com/apache/hadoop/blob/58fbe4a955497d240639e5f0205fa25c3ebb780e/hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystemStore.java#L1262-L1274), It seems that they just generate a `continuation` by hand.